index.js

```diff
@@ -25,13 +26,31 @@ function Level (location, opts) {
   this.location = location
   this.prefix = opts.prefix || DEFAULT_PREFIX
   this.version = parseInt(opts.version || 1, 10)
+
+  // Experimental, do not externally rely on this object yet.
+  // See Level/community#42.
+  this.supports = {
+    bufferKeys: Level.binaryKeys,
+    binaryKeys: Level.binaryKeys,
+    keys: {
+      string: true,
+      buffer: Level.binaryKeys,
+      arrayBuffer: Level.binaryKeys,
+      typedArray: Level.binaryKeys
+    },
```

Comment:
What do we prefer?

1. Have both `bufferKeys` and `binaryKeys` properties, where the latter means any binary type (ArrayBuffer and views)
1. Only have `bufferKeys` property, because in the ecosystem we only have Buffer atm
1. Have a `keys` object that indicates support of specific types
1. Don't support ArrayBuffer and typed arrays at all (for now)

@ralphtheninja

Reply:
For the manifest in general I prefer option 3, it's future proof.

Whether to support ArrayBuffer and typed arrays here, I can't decide. Sure it's nice, and doesn't complicate `level-js` too much, but the Level ecosystem only supports Buffers. That might be a good thing, for simplicity sake. E.g. I don't really want to deal with ArrayBuffer and typed arrays in `subleveldown`.

Reply:
Using Buffer as the one and only binary type does increase the browserify/webpack bundle size. 

IMO that's negated by the rest of the ecosystem (including the dependencies of `level-js` itself) not needing extra code to handle ArrayBuffer and typed arrays.